.circleci/config.yml

```diff
@@ -724,7 +724,7 @@ jobs:
       - attach_workspace:
           at: .
       - steps: << parameters.optional_steps >>
-      - browser-tools/install-browser-tools
```

Comment:
This installs several browser tools, but we only need Chrome.

The problem was in Chrome**driver** which is a separate install.